### ISSUE TITLE
The rack.monitor not loaded

### ISSUE THREAD

**Issue by elct9620** [elct9620/hanami-lambda#2]
The `rack.monitor` isn't loaded when Lambda calls the event `Hanami.boot` is called.

**Comment by elct9620** [comment]
The reason is `rack` gem is not loaded correctly.

https://github.com/hanami/hanami/blob/73adf779ed5a75ecfe576cd8f8c1d034791f5102/lib/hanami/app.rb#L160-L163

```
        if Hanami.bundled?("rack")
          require_relative "providers/rack"
          register_provider(:rack, source: Hanami::Providers::Rack, namespace: true)
        end
```

**Comment by elct9620** [comment]
Fixed by 16fbce66514c32b8cb23201a2651d9fa8d56bedf
